projects/plugins/jetpack/_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -847,7 +847,7 @@ const features = {
 				checked: getSetting( state, 'seo-tools' ),
 				configureLink,
 				upgradeLink,
-				isPaid: true,
+				isPaid: false,
```

Comment:
You should now be able to get rid of all the plan checking logic above this.

Reply:
89d17d72387312ae325a547c78dcf7e0a4c5d922